src/main/java/calculator/CalculatorApplication.java

```diff
@@ -0,0 +1,17 @@
+package calculator;
+
+import java.util.Scanner;
+
+public class CalculatorApplication {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        CalculatorController calculatorController = new CalculatorController();
+
+        System.out.println("Enter the input: ");
+        String input = scanner.nextLine();
+        System.out.println("Result: " + calculatorController.add(input));
+
+        scanner.close();
```

Comment:
아래와 같이 `Scanner` 같은 `Closeable` 구현체는 [try with resource](https://www.baeldung.com/java-try-with-resources)를 활용하도록 변경해볼 수 있을 것 같아요. 😃

```suggestion
        try(Scanner scanner = new Scanner(System.in)) {
            CalculatorController calculatorController = new CalculatorController();

            System.out.println("Enter the input: ");
            String input = scanner.nextLine();
            System.out.println("Result: " + calculatorController.add(input));
        } catch (Exception e) {
            System.out.println("Error: " + e.getMessage());
        }
```